### PR TITLE
Wfnews 814

### PR DESCRIPTION
**Pull request by yzlucas**
Add fireCentreName attribute and update api to fetch and orderBy it. I have reviewed with Sean to verify the validation

### CODE REVIEW

File: server/wfnews-persistence/src/main/java/ca/bc/gov/nrs/wfnews/persistence/v1/dto/PublishedIncidentDto.java [hunk 1]
```diff
@@ -108,6 +109,7 @@ public PublishedIncidentDto(PublishedIncidentDto dto) {
 		this.longitude = dto.longitude;
 		this.declaredOutDate = dto.declaredOutDate;
 		this.fireCentre = dto.fireCentre;
+		this.fireCentreName = dto.fireCentreName;
```
Comment (dhlevi) [inline server/wfnews-persistence/src/main/java/ca/bc/gov/nrs/wfnews/persistence/v1/dto/PublishedIncidentDto.java:112]: You added the attribute here, but not in the resource constructor. Add it to the constructor that passes in a PublishedIncident object (line 118)

Comment (dhlevi) [reply server/wfnews-persistence/src/main/java/ca/bc/gov/nrs/wfnews/persistence/v1/dto/PublishedIncidentDto.java:112]: You also didn't add a check in the `equalsAll` method, which is required for equality checks.

**Review by dhlevi — CHANGES_REQUESTED** [review]
You're missing mappings in PublishedIncidentResourceFactory, which would prevent the new attribute from ever being mapped? Make sure you test API changes, there are Docker scripts set up to assist with this if you don't have a local debug environment for java APIs.

File: database/scripts/01_00_00/05/ddl/tables/MODIFY.PUBLISHED_INCIDENT_DETAIL.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+-- Add FIre Centre Name
+ALTER TABLE "wfnews"."published_incident_detail" ADD COLUMN IF NOT EXISTS "fire_centre_name" VARCHAR(120);
+COMMENT ON COLUMN "wfnews"."published_incident_detail"."fire_centre_name" IS 'fire_centre_name is the name of fire centre of the incident'
```
Comment (dhlevi) [inline database/scripts/01_00_00/05/ddl/tables/MODIFY.PUBLISHED_INCIDENT_DETAIL.sql:3]: The comment on the existing column `"wfnews"."published_incident_detail.fire_centre"` should be updated to reflect that it's only storing the Code. Additionally, now that we have a fire_centre_name, column, that old column should be renamed to fire_centre_code or something sensible to reflect what it now is.

**Comment by yzlucas** [comment]
> You're missing mappings in PublishedIncidentResourceFactory, which would prevent the new attribute from ever being mapped? Make sure you test API changes, there are Docker scripts set up to assist with this if you don't have a local debug environment for java APIs.

**Pull request closed without merging**